use-cases/accounting.md

```diff
@@ -28,6 +28,14 @@ Before assisting any new client with connecting to your accounting integration,
 
 Ensure you follow our general [guidelines](../guidelines) for testing integrations.
 
+### Get and query rebated items
+
+You can use the API to find out if an accounting item is rebated. First use [Get all bills](../operations/finance.md#get-all-bills) to get a list of the order items on the bill. For each [order item](../operations/finance.md#order-item), examine the order item data to identify those items which are Rebates and if so, what is the [Rebated item ID](../operations/finance.md#rebate-order-item-data). You can then call [Get all accounting items](../operations/finance.md#get-all-accounting-items) with the Rebated item ID to return the details of the accounting item with that ID.
```

Comment:
The first sentence says `You can use the API to find out if an accounting item is rebated.` But then the steps don't actually help you do that. You need to: 
- Take an Item
  - Don't care if it's a rebate, because the question is what rebates this, not what does this rebate.
- Put the ID of the item into the `RebatedItemId` of the accountingItems/Get
- Get collection of Rebates
- Either find a rebate for it, or conclude that it's not rebated yet.


You're answering a different question `If I get a rebate on a bill, what should I do to get the original item` which is a different question.

Reply:
By the way, I still think that this is a useful paragraph. It's just answering a different question. We should keep both. 
```suggestion
You can use the API to find out if an accounting item is rebated. First use [Get all bills](../operations/finance.md#get-all-bills) to get a list of the order items on the bill. For each [order item](../operations/finance.md#order-item), examine the order item data to identify those items which are Rebates and if so, what is the [Rebated item ID](../operations/finance.md#rebate-order-item-data). You can then call [Get all accounting items](../operations/finance.md#get-all-accounting-items) with the Rebated item ID to return the details of the accounting item with that ID. This will give you more info on the original item that was rebated.
```